docs/release-notes/cloud/cloud-2021-12.md

```diff
@@ -0,0 +1,79 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud December 2021.
+included_jira_versions: ['2021.Q4.6', '2021.Q4.7']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/4.0.22
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/4.0.54
+---
+
+# Cloud December 2021
+
+These release notes are for the Codacy Cloud updates during December 2021.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   It's now possible to [assign a coding standard automatically to new repositories](https://docs.codacy.com/organizations/using-a-coding-standard/#set-default). (CY-5263)
+
+## Bug fixes
+
+-   Fixed an issue that could cause payment plans to change to "Open source" when <span class="skip-vale">Chargebee</span> is unreachable. (CY-5386)
```

Comment:
what is this "skip-vale" I don't see any difference visually in the preview 👀

Reply:
This is just an inline way of ignoring certain words on the [Vale linter](https://github.com/errata-ai/vale) to avoid false positives.

If the words are frequent I normally add them to the [accept list](https://github.com/codacy/docs/blob/master/.github/styles/Vocab/Codacy/accept.txt), otherwise, I prefer to just ignore them like this.